test/libcommonserver/io/testchecksetgetrights.cpp

```diff
@@ -75,9 +75,9 @@ void TestIo::testCheckSetAndGetRights() {
         IoHelper::_getAndSetRightsMethod = 0; // Set the method to use the windows API
         for (int i = 0; i < 2; i++) { // Test Windows API and fallback method
 #endif
-            for (int baseRigths = 0; baseRigths < 7; baseRigths++) {
-                for (int targetRigths = baseRigths + 1; targetRigths < 8; targetRigths++) {
-                    auto rightsSet = RightsSet(baseRigths);
+            for (int baseRights = 0; baseRights < 7; baseRights++) {
```

Comment:
Spaces versus tabs?

Reply:
"Rigths" => "Rights"

Reply:
Oh, I didn't see the difference. I certainly need a TDAH check-up!